app.js

```diff
@@ -2,29 +2,18 @@ const express = require("express");
 const app = express();
 const healthcheck = require("./controllers/healthcheck.controller");
 const { getTopics } = require("./controllers/topics.controller");
-// const {} = require("./controllers/articles.controller");
-// const {} = require("./controllers/comments.controller");
-// const {} = require("./controllers/users.controller");
+const endpointsList = require("./endpoints.json");
 
 app.use(express.json());
 
-//Endpoints:
-
 app.get("/api/healthcheck", healthcheck);
 
 app.get("/api/topics", getTopics);
 
-// Error handling:
-
-app.use((err, request, response, next) => {
-  if (err.status) {
-    response.status(err.status).send({ msg: err.msg });
-  }
-  next(err);
+app.get("/api", (request, response) => {
+  response.status(200).send({ endpoints: endpointsList });
```

Comment:
this line needs to be part of a controller ! :) no model needed but we do need a controller!